remo/annotation_utils.py

```diff
@@ -20,7 +20,7 @@ def parse_csv_obj_det(file_path) -> List[Annotation]:
         List[:class:`remo.Annotation`]
     """
 
-    annotations = []
+    annotations_list = []
```

Comment:
@drewlr I don't see reason rename `annotations` to `annotations_list`

Reply:
@vmanilo yes this we could leave "annotations"...I just find "annotation" and "annotations" very similar to each other, so I thought "annotations_list" makes it more obvious